clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -38,11 +38,7 @@ void dumpExtFuncMap(const ProgramInfo::ExternalFunctionMapType &EMap,
                     raw_ostream &O) {
   for (const auto &DefM : EMap) {
     O << "Func Name:" << DefM.first << " => ";
-    for (const auto J : DefM.second) {
-      O << "[ ";
-      J->print(O);
-      O << " ]\n";
-    }
+    DefM.second->print(O);
```

Comment:
But the old code would have printed brackets, even for a singleton; your new code is not doing that?